bears/php/PHPStanBear.py

```diff
@@ -0,0 +1,21 @@
+from coalib.bearlib.abstractions.Linter import linter
+
+
+@linter(executable='phpstan',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+)  (?P<message>.*)')
+class PHPStanBear:
+    """
+    Checks the code with ``phpstan analyze``.
+    This can run it on multiple files and folders.
+    See <https://github.com/phpstan/phpstan> for more information.
+    """
+    LANGUAGES = {'PHP'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Syntax'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        return ('analyse', '--no-ansi', filename)
```

Comment:
Also regarding `rule-level`, try running `phpstan` directly with different levels and see if there is any output or if the regex changes. In your case, you've set the default level to 0 (which is the loosest), so I reckon that's why it wasn't able to capture your bad file errors.